docs/data/toolpad/core/introduction/tutorial.md

```diff
@@ -1,3 +1,143 @@
-# Tutorial
+---
+title: Tutorial
+---
+
+# Toolpad Core - Tutorial
 
 <p class="description">Tutorial</p>
+
+## Steps to follow
```

Comment:
Can we format the tutorial with sections, instead of a numbered list? That way we get navigation on the right.

Perhaps it makes sense to have two sections?
```
## bootstrapping the application

commands to get started

## creating a page

bootstrapping 1 page + making it appear in navigation
```
Then this is where is where we can continue with https://deploy-preview-3611--mui-toolpad-docs.netlify.app/toolpad/core/introduction/dashboard-tutorial/